site/_resources/kubernetes.md

```diff
@@ -11,13 +11,16 @@ The `client-go` package includes a [compatibility matrix][2] as to what Kubernet
 
 ## Supported Kubernetes versions
 
-| Kubernetes version | Contour v1.0.0 | Contour v1.0.1 | Contour v1.1.0 |
-| ------------ | :-----------: | :-----------: | :----------: | 
-| 1.15.x | Supported | Supported | Supported | 
-| 1.16.x | Supported | Supported | Supported |
-| 1.17.x | Supported | Supported | Supported | 
-| 1.18.x | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup> | Not Supported <sup>1</sup> |
+| Kubernetes version | Contour v1.0.0 | Contour v1.0.1 | Contour v1.1.0 | Contour v1.2.0 |
+| ------------ | :-----------: | :-----------: | :----------: | :--------: |
+| 1.13.x | Supported | Supported | Supported | Not Supported<sup>1</sup> |
+| 1.14.x | Supported | Supported | Supported | Not Supported<sup>1</sup> |
+| 1.15.x | Supported | Supported | Supported | Supported |
+| 1.16.x | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup> | Supported |
+| 1.17.x | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup> | Supported |
+| 1.18.x | Not Supported<sup>1</sup>  | Not Supported<sup>1</sup> | Not Supported <sup>1</sup> | Not Supported <sup>1</sup> |
```

Comment:
I'd prefer that we don't list versions that don't exist yet, then we don't have to say if they are supported or not. It's a Heisenberg thing.